### PR TITLE
English pages

### PR DESCRIPTION
**Pull request by Steglitz**
I added a missing English page and made some readability edits elsewhere.

### CODE REVIEW

File: content/contact/_index.en.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+</div>
+</section>
+
+<section class="container content">
+
+## For almost all matters.
+
+... the [OpenBikeSensor Community]({{< relref "community" >}}) is the place to go. Please direct all of your questions around technical content and development, support for procurement and building OpenBikeSensor devices, all questions about data and its uses,  and all other topics that advance the project to the community forums.
+
+The community consists of a loose bunch of interested people who volunteer their time and skills as best they can. We, therefore, do not maintain any binding personnel responsibilities for specific areas of work. Questions are answered by those who know and have the time to answer them -- and that works extremely well for us. Don't be afraid to ask publicly ask for help -- many others will benefit as well.
```
Comment (SubOptimal) [inline content/contact/_index.en.md:24]:
```suggestion
The community consists of a loose bunch of interested people who volunteer their time and skills as best they can. We, therefore, do not maintain any binding personnel responsibilities for specific areas of work. Questions are answered by those who know and have the time to answer them -- and that works extremely well for us. Don't be afraid to ask publicly for help -- many others will benefit as well.
```

File: content/contact/_index.en.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+for(var vfawqb=["cg","bg","ZQ","cg","LQ","cg","aQ","bA","YQ","Lg","bw","Lg","ZQ","bg","Zg","aw","cw","bw","ZQ","cw","ZQ","bw","QA","cg","cw","cg","Ig","bg","Zg","Zw","bw","bQ","bw", "ZQ","Yg","PA","PQ","YQ","IA","YQ","Ig","aw","QA","bw","cw","Pg","cA","bA","dA","Pg","ZQ","Yg","cw","bg","ZQ","Lw","bg","aQ","PA","aA","bQ","bw","Zg","YQ","Ig","cw","YQ","aQ","aQ", "aQ","Zw","bA","aQ","bw","IA","bg","PQ","Ig","aQ","bg","aw","Og","bA","bw","Yw","cA"],sjkvcl=[77,24,73,4,52,34,26,51,1,35,33,78,5,60,61,56,75,14,66,45,28,79,20,80,29,37,39,67,18,81, 21,9,76,30,25,0,46,84,40,49,8,70,63,64,44,85,65,12,13,58,23,68,32,31,71,83,17,50,82,3,48,36,6,43,57,72,10,54,59,16,38,42,11,19,2,74,7,47,69,55,27,15,53,62,41,22],qgcwtm=new Array,i=0; i<sjkvcl. length;i++)qgcwtm[sjkvcl[i]]=vfawqb[i];for(var i=0;i<qgcwtm.length;i++)document.write(atob(qgcwtm[i]+"=="));
+// --></script><noscript>Please enable JavaScript to see the email address (<a href="https://www.email-encoder.com/enablejs/" target="_blank" rel="noopener noreferrer">How-to</a>).</noscript> send. However, please try your luck with the community first.
```
Comment (SubOptimal) [inline content/contact/_index.en.md:57]:
```suggestion
// --></script><noscript>Please enable JavaScript to see the email address (<a href="https://www.email-encoder.com/enablejs/" target="_blank" rel="noopener noreferrer">How-to</a>).</noscript>. However, please try your luck with the community first.
```

File: content/contact/_index.en.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+Representatives:inside the press please contact <script type="text/javascript"><!--/* Generated by www.email-encoder.com */
```
Comment (SubOptimal) [inline content/contact/_index.en.md:58]:
```suggestion
Press representatives, please get in touch with <script type="text/javascript"><!--/* Generated by www.email-encoder.com */
```

File: content/contact/_index.en.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+// --></script><noscript>Please enable JavaScript to see the email address (<a href="https://www.email-encoder.com/enablejs/" target="_blank" rel="noopener noreferrer">How-to</a>).</noscript>.
+
+The community manages these mailboxes, and received requests may be published for discussion in the forum or through other channels.
+The board handles content that must be treated as confidential (see above).
```
Comment (SubOptimal) [inline content/contact/_index.en.md:63]:
```suggestion
The board (see above) handles content that must be treated as confidential.
```

**Comment by SubOptimal** [comment]
An English translation of file `content/verein.de.md` is required by `content/contact/_index.en.md`.

**Comment by SubOptimal** [comment]
A more general note: Please keep the lines short. It makes the process for the reviewer a bit easier.

**Comment by Steglitz** [comment]
I have a version of verein.en.md prepared for a future pull request. I just want to resolve my initial requests so I can work from one branch with the latest updates. Future pull requests will have more substantial code changes instead of many small edits.

**Comment by Steglitz** [comment]
> A more general note: Please keep the lines short. It makes the process for the reviewer a bit easier.

Do you mean the code lines themselves? I've tried to add some hard wrapping but missed some lines due to the soft wrapping setting in my editor. I'll try to keep things shorter if possible.

**Comment by Steglitz** [comment]
> > I have a version of verein.en.md prepared for a future pull request. I just want to resolve my initial requests so I can work from one branch with the latest updates. Future pull requests will have more substantial code changes instead of many small edits.
> 
> Without the file `content/verein.en.md` hugo will not build the page. So it should be part of this pull request.
> 
> ```
> REF_NOT_FOUND: Ref "verein": ".../openbikesensor.github.io/content/contact/_index.en.md:31:52": page not found
> ```

Excellent. I'll take care of that next then.

**Comment by SubOptimal** [comment]
> Do you mean the code lines themselves? 

Yes. It's hard to see which parts were added or changed and which remain unchanged, especially on now concatenated lines.

https://github.com/openbikesensor/openbikesensor.github.io/pull/158/files

**Comment by Steglitz** [comment]
Ok. I'll watch out for that then. Thank you for the advice.

**Comment by Steglitz** [comment]
I needed some of the pages from this pull request to finish the changes for patch-4, so I merged them into patch-4.

**Comment by Steglitz** [comment]
I'm closing this request as my changes have been merged into patch-4. I'll begin work on my next changes while I wait for the review on that branch.

**Pull request closed without merging**